libraries/Microsoft.Bot.Connector/Attachments.cs

```diff
@@ -66,152 +62,168 @@ public Attachments(ConnectorClient client)
         /// The cancellation token.
         /// </param>
         /// <exception cref="ErrorResponseException">
-        /// Thrown when the operation returned an invalid status code
+        /// Thrown when the operation returned an invalid status code.
         /// </exception>
         /// <exception cref="SerializationException">
-        /// Thrown when unable to deserialize the response
+        /// Thrown when unable to deserialize the response.
         /// </exception>
         /// <exception cref="ValidationException">
-        /// Thrown when a required parameter is null
+        /// Thrown when a required parameter is null.
```

Comment:
>Thrown when a required parameter is null. [](start = 12, length = 41)

This is thrown only when attachment Id is null. If you update this comment, you can remove the pragma that disables SA1625